src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        String color = supplier.getRandomColor();
+        int num = new Random().nextInt(100);
+
+        return new Ball(color, num);
```

Comment:
You're creating a new instance of Random every time the getRandomBall method is called. It's better to create a single instance of Random as a class member and reuse it.